src/create_disk.sh

```diff
@@ -79,18 +88,54 @@ sgdisk -Z $disk \
 	-n 4:0:0     -c 4:root       -t 4:4F68BCE3-E8CD-4DB1-96E7-FBCAF984B709
 sgdisk -p "$disk"
 
-udevadm trigger
-udevadm settle
+udevtrig
+
+root_dev="${disk}4"
+if [ -n "${luks_rootfs}"  ]; then
+    root_dev=/dev/mapper/crypt_root
+    sgdisk -c 4:luk_root "${disk}"
+
+    touch tmp.key
+    # Create the LUKS partition using the null_cipher and a sentinal
+    # UUID similiar to the one used by coreos-gpt-setup. This is used
+    # by ignition-dracut-reecrypt.
+    cryptsetup luksFormat \
+        -q \
+        --label="crypt_rootfs" \
+        --cipher=cipher_null \
+        --key-file=tmp.key \
+        --uuid='00000000-0000-4000-a000-000000000002' \
+        "${disk}4"
+
+    echo "" | cryptsetup luksOpen \
```

Comment:
This acts as both a test and documentation that you can open the LUKS volume with no password.